portal/views/portal.py

```diff
@@ -141,6 +141,8 @@ def report_slow_queries(response):
 
     """
     if current_app.config.get("LOG_SLOW_RESPONSES"):
+        if not hasattr(g, 'start_request_time'):
+            return response
```

Comment:
This doesn't look related to the bugfix. If I understand correctly, this seems to exit early if the request is not being timed, but isn't that controlled by the above config?

Reply:
it's just something that blew up on me when testing - never seen it before, but seemed worthy of adding a safeguard.